js/glossary.js

```diff
@@ -127,13 +128,13 @@ Glossary.prototype.show = function(e) {
 Glossary.prototype.hide = function() {
   this.$body.removeClass('is-open').attr('aria-hidden', 'true');
   this.$toggle.removeClass('active');
-  this.$trigger.focus();
+  this.$trigger && this.$trigger.focus();
```

Comment:
So only trigger focus if the trigger exists. I like this pattern, maybe we should relax the houndci warnings?